winui/Radial-Gauge/content-pointer.md

```diff
@@ -0,0 +1,206 @@
+    CornerRadius = new CornerRadius { BottomLeft = 4, BottomRight = 4, TopLeft = 4, TopRight = 4 }
+};
+contentPointerRootChild.Children.Add(new TextBlock
+{
+    Text = "Syncfusion",
+    Margin = new Thickness { Bottom = 2, Left = 2, Right = 2, Top = 2 }
+});
+
+contentPointer.Content = contentPointerRootChild;
+radialAxis.Pointers.Add(contentPointer);
+
+this.Content = sfRadialGauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![WinUI Radial Gauge Content Pointer](images/content-pointer/default-content-pointer.png)
+
+## Add Image to Content Pointer
+
+The content pointer of `SfRadialGauge` can also have an image as its `Content`.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<gauge:SfRadialGauge>
+    <gauge:SfRadialGauge.Axes>
+        <gauge:RadialAxis>
+            <gauge:RadialAxis.Pointers>
+                <gauge:ContentPointer Value="60">
+                    <gauge:ContentPointer.Content>
+                        <Grid Background="Orange"
+                              BorderBrush="Black"
+                              BorderThickness="1"
+                              CornerRadius="4">
+                            <Image Source="pin.png" />
+                        </Grid>
+                    </gauge:ContentPointer.Content>
+                </gauge:ContentPointer>
+            </gauge:RadialAxis.Pointers>
+        </gauge:RadialAxis>
+    </gauge:SfRadialGauge.Axes>
+</gauge:SfRadialGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfRadialGauge sfRadialGauge = new SfRadialGauge();
+
+RadialAxis radialAxis = new RadialAxis();
+sfRadialGauge.Axes.Add(radialAxis);
+
+ContentPointer contentPointer = new ContentPointer();
+contentPointer.Value = 60;
+contentPointer.IsInteractive = true;
+Grid contentPointerRootChild = new Grid
+{
+    Background = new SolidColorBrush(Colors.Orange),
+    BorderBrush = new SolidColorBrush(Colors.Black),
+    BorderThickness = new Thickness { Bottom = 1, Left = 1, Right = 1, Top = 1 },
+    CornerRadius = new CornerRadius { BottomLeft = 4, BottomRight = 4, TopLeft = 4, TopRight = 4 }
+};
+contentPointerRootChild.Children.Add(new Image
+{
+    Source = "pin.png",
+    Margin = new Thickness { Bottom = 2, Left = 2, Right = 2, Top = 2 }
+});
+
+contentPointer.Content = contentPointerRootChild;
+radialAxis.Pointers.Add(contentPointer);
+
+this.Content = sfRadialGauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![WinUI Radial Gauge Content Pointer with Image](images/content-pointer/winui-radial-gauge-image-pointer.png)
+
+## Content position customization
+
+The content pointer can be moved near or far from its actual position using the `MarkerOffset` and `OffsetUnit` properties. 
+
+When you set `OffsetUnit` to pixel, the content pointer will be moved based on the pixel value. If you set `OffsetUnit` to factor, then provided factor will be multiplied with the axis radius value, and then the pointer will be moved to corresponding value. The default value of `OffsetUnit` is `SizeUnit.Pixel`.
```

Comment:
We have API links for OffsetUnit and SizeUnit.

Reply:
API link for OffsetUnit and SizeUnit was added.